global.json

```diff
@@ -1,11 +1,11 @@
 {
   "tools": {
-    "dotnet": "2.2.203"
+    "dotnet": "3.0.100-preview5-011568"
```

Comment:
@mmitche is it normal to have arcade update the sdk used by my repo? This is causing build issues as some of my projects target netcoreapp2.1 which apparently doesn't come as part of this sdk.

Reply:
Yes, this is intentional.

Please see: https://github.com/dotnet/arcade/blob/master/Documentation/ArcadeSdk.md#example-restoring-multiple-net-core-runtimes-for-running-tests

You should also have gotten a mail about this yesterday

/cc @riarenas

Reply:
@joperezr Does adding the 2.1 runtime fix your issues?

You can add it to your global.json as seen here: https://github.com/dotnet/arcade-validation/blob/70441d054cf3f57c3d7b5a6960a3092bdae58d56/global.json#L4 ?

Reply:
just pushed a change to this PR with that, I'll see if that fixes the issue. Thanks!